content/sensu-go/5.4/reference/checks.md

```diff
@@ -93,7 +72,219 @@ At every execution of a check command – regardless of success or failure – t
 Sensu agent publishes the check’s result for eventual handling by the **event
 processor** (the Sensu backend).
 
-### Check token substitution
+## Check scheduling
+
+Checks are scheduled by the Sensu backend, which schedules and
```

Comment:
```suggestion
Checks are scheduled by the Sensu backend, which 
```

Reply:
Included this change in https://github.com/sensu/sensu-docs/pull/1339/commits/8118a778485d0d33e288553c53870d69013a8eea